tests/rptest/services/tls.py

```diff
@@ -534,18 +533,32 @@ def _create_ca(self,
 
         return CertificateAuthority(cfg, key, crt, crl)
 
-    def _create_ca_cert_chain(self, files: list[str]) -> CertificateAuthority:
+    def _create_ca_cert_chain(self) -> CertificateAuthority:
+        # First create the signing ca chain
+        ca_files = [ca.crt for ca in self._cas]
         out = self._with_dir('ca', 'signing-ca-chain.pem')
         pathlib.Path(out).touch()
         with open(out, 'w') as outfile:
-            for fname in reversed(files):
+            for fname in reversed(ca_files):
                 with open(fname, 'r') as infile:
                     outfile.write(infile.read())
 
         with open(out, 'r') as f:
             self._logger.debug(f"CA chain: {f.read()}")
 
-        return CertificateAuthority(None, None, out, self._cas[-1].crl)
+        # Now do the same for the CRLs
+        crl_files = [ca.crl for ca in self._cas]
+        crl_out = self._with_dir('ca', 'signing-crl-chain.crl')
+        pathlib.Path(out).touch()
```

Comment:
actually, did you mean

```suggestion
        pathlib.Path(crl_out).touch()
```

Reply:
doesn't look like it matters, so I'm not sure what it was doing there previously 🤷